### ISSUE TITLE
watch=true in config file is not respected, but works when added as a command argument

### ISSUE THREAD

**Issue by zytek** [kelseyhightower/confd#242]

**Comment by zytek** [comment]
```
/etc/confd$ cat confd.toml
backend = "etcd"
nodes = [
  "http://localhost:4001"
  ]
verbose = true
watch = true
debug = true
13:37:02 prompt
/etc/confd$ sudo confd
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: NOTICE Backend set to etcd
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: NOTICE Starting confd
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: NOTICE Backend nodes set to http://localhost:4001
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Loading template resources from confdir /etc/confd
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Loading template resource from /etc/confd/conf.d/main.toml
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Retrieving keys from store
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Key prefix set to /databases/main
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Using source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Compiling source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Comparing candidate config to /etc/haproxy/databases/main.conf
2015-03-30T13:37:05Z ip-10-105-4-94 confd[21141]: DEBUG Target config /etc/haproxy/databases/main.conf in sync

[updated entry - no changes]

^C
2015-03-30T13:37:15Z ip-10-105-4-94 confd[21141]: INFO captured interrupt exiting...

13:37:15 prompt
/etc/confd$ sudo confd -watch=true
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: NOTICE Backend set to etcd
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: NOTICE Starting confd
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: NOTICE Backend nodes set to http://localhost:4001
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Loading template resources from confdir /etc/confd
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Loading template resource from /etc/confd/conf.d/main.toml
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Retrieving keys from store
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Key prefix set to /databases/main
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Using source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Compiling source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Comparing candidate config to /etc/haproxy/databases/main.conf
2015-03-30T13:37:18Z ip-10-105-4-94 confd[21148]: DEBUG Target config /etc/haproxy/databases/main.conf in sync
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Retrieving keys from store
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Key prefix set to /databases/main
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Using source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Compiling source template /etc/confd/templates/database.tmpl
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Comparing candidate config to /etc/haproxy/databases/main.conf
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: INFO /etc/haproxy/databases/main.conf has md5sum e16357e585586badb7a86a3d70e40445 should be f93f67b55323893d031ff12cf6205a8f
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: INFO Target config /etc/haproxy/databases/main.conf out of sync
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Overwriting target config /etc/haproxy/databases/main.conf
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG Running /etc/haproxy/reload_wrapper reload main /etc/haproxy/databases/main.conf
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: DEBUG ""
2015-03-30T13:37:24Z ip-10-105-4-94 confd[21148]: INFO Target config /etc/haproxy/databases/main.conf has been updated
^C2015-03-30T13:37:27Z ip-10-105-4-94 confd[21148]: INFO captured interrupt exiting...
```

**Comment by bacongobbler** [comment]
see #236

**Comment by zytek** [comment]
@bacongobbler is this really the case?

config file IS loaded as you can see by confd printing debug messages - becase of debug=true in conf file. If I move it away than there is no debug messages.

I am using 0.7.1

**Comment by bacongobbler** [comment]
Yes. I closed this because it's a duplicate of #208.
